### PR TITLE
Mc us 36

### PR DESCRIPTION
**Pull request by Mariana-21**
## All Submissions:

* [X] Merge the target branch into the PR branch. Fix any conflicts that might appear.
* [X] Have you checked to ensure that other pull request are approved for merge [Pull Requests](../../../pulls)?
* [ ] What stories does the PR include
userstory 36
### New Feature Submissions:

* [X] Have you added an explanation of what your changes do?
* [X] Have you lint your code locally prior to submission?

### Testing:

* [X] Have you written new tests for your core changes, as applicable?
* [X] Have you successfully ran tests with your changes locally?
* [X] All new and existing tests passed.

**Test coverage** = 100% features/models
--------------------------------------------------------------------------------
### Change Log

* Created A new test spec to assert link is on dashboard for merchant
* Created link in the merchant dashboard
* Created /merchant/items route
* Created Merchant::ItemsController
* Created Merchant/items/index.html.erb page

### CODE REVIEW

File: spec/features/merchant/items_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  end
+end
+
+
+
+# As a merchant employee
+# When I visit my merchant dashboard
+# I see a link to view my own items
+# When I click that link
+# My URI route should be "/merchant/items"
```
Comment (tylertomlinson) [inline spec/features/merchant/items_index_spec.rb]: Can this be removed

File: app/controllers/merchant/items_controller.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Merchant::ItemsController < ApplicationController
```
Comment (tylertomlinson) [inline app/controllers/merchant/items_controller.rb:1]: Nice job with the hierarchy here.

File: config/routes.rb [hunk 3]
```diff
@@ -27,6 +27,7 @@
 
   namespace :merchant do
     get "/", to: "dashboard#index"
+    get "/items", to: "items#index"
```
Comment (tylertomlinson) [inline config/routes.rb:30]: I think moving forward we should use resources. Simple fix so don't worry now

**Review by tylertomlinson — APPROVED** [review]

File: spec/features/merchant/items_index_spec.rb (hunk 1 above)
Comment (Mariana-21) [reply spec/features/merchant/items_index_spec.rb]: All gone now

File: config/routes.rb (hunk 3 above)
Comment (Mariana-21) [reply config/routes.rb:30]: Yeah, I agree we should change that eventually

**Review by alex-latham — APPROVED** [review]
These changes look good to me

**Pull request merged**